packages/vx-demo/src/pages/BarStack.tsx

```diff
@@ -87,13 +86,16 @@ export default withTooltip<ShowProvidedProps, TooltipData>(
       bottom: 0,
       left: 0,
     },
-    tooltipOpen,
-    tooltipLeft,
-    tooltipTop,
-    tooltipData,
-    hideTooltip,
-    showTooltip,
-  }: ShowProvidedProps & WithTooltipProvidedProps<TooltipData>) => {
+  }: ShowProvidedProps) => {
+    const {
```

Comment:
This stuck out to me as something to come back to as well.  It might be doable with `raw-loader` or [`raw.macro`](https://www.npmjs.com/package/raw.macro) but I didn't have time to dig into it more deeply.  Now this has me curious so I'll look into it later in the week if I have a chance 🤔

Reply:
sounds good! Created #632 to track this, I'll update status there if I start work on it. `raw.macro` looks interesting, linked to how `react-window` does it in the issue as well 👍